ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13.yaml

```diff
@@ -0,0 +1,46 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-aws-413
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-kube-burner
+  cron: 29 5 */1 * 1-5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      WORKER_REPLICA_COUNT: "20"
```

Comment:
Do we want to try to mimic the managed offering on our node sizes?

Reply:
yes, we have defined "profiles" we used in jenkins and will probably want to try to match that set up here